pre-docs/bolt_command_reference.md

```diff
@@ -1,109 +1,498 @@
+| **Run Context** |
+| `-c`, `--concurrency CONCURRENCY` | Maximum number of simultaneous connections. | 100 |
+| `-m`, `--modulepath FILEPATHS` | List of directories containing modules, separated by `:`. <br> Directories are case-sensitive. |
+| `--boltdir FILEPATH` | Specify what Boltdir to load config from. | Autodiscovered from current working directory. |
+| `--configfile FILEPATH` | Specify where to load config from. | `~/.puppetlabs/bolt/bolt.yaml` |
+| `-i`, `--inventoryfile FILEPATH` | Specify where to load inventory from. | `~/.puppetlabs/bolt/inventory.yaml` |
+| `--[no-]save-rerun` | Whether to update the rerun file after this command. |
+| **Transports** |
+| `--transport TRANSPORT` | Specify a default transport. <br> `ssh`, `winrm`, `pcp`, `local`, `docker`, `remote` |
+| `--connect-timeout TIMEOUT` | Connection timeout. | Varies |
+| `--[no-]tty` | Request a pseudo TTY on nodes that support it. |
+| `--tmpdir DIR` | The directory to upload and execute temporary files on the target. |
+| **Display** |
+| `--format FORMAT` | Output format to use. <br> `human`, `json` |
+| `--[no-]color` | Whether to show output in color. |
+| `-v`, `--[no-]verbose` | Display verbose logging. |
+| `--trace` | Display error trace stacks. |
+
+
+## `inventory show`
+
+Show the list of targets an action would run on.
+
+### Usage
+
+`bolt inventory show <TARGETS>`
+
+- You must specify one of `--nodes`, `--targets`, `--query`, or `--rerun`.
+
+### Options
+
+| Option | Description | Default |
+|--------|-------------|---------|
+| `-n`, `--nodes NODES` | Alias for `--targets`. |
+| `-t`, `--targets TARGETS` | Identifies the targets of command. |
+| `-q`, `--query QUERY` | Query PuppetDB to determine the targets. <br> Enter a comma-separated list of target URIs or group names. Or read a target list from an input file `@<file>` or stdin `-`. |
+| `--rerun FILTER` | Retry on nodes from the last run. <br> `all` runs on all targets from the last run. <br> `failure` runs on all targets that failed in the last run. <br> `success` runs on all targets that succeded in the last run. |
+| **Run Context** |
+| `--boltdir FILEPATH` | Specify what Boltdir to load config from. | Autodiscovered from current working directory. |
+| `--configfile FILEPATH` | Specify where to load config from. | `~/.puppetlabs/bolt/bolt.yaml` |
+| `-i`, `--inventoryfile FILEPATH` | Specify where to load inventory from. | `~/.puppetlabs/bolt/inventory.yaml` |
+| **Display** |
+| `--format FORMAT` | Output format to use. <br> `human`, `json` |
+
+
+## `plan convert`
+
+Convert a YAML plan to a Puppet plan.
+
+### Usage
+
+`bolt plan convert <PLAN>`
+
+### Options
+
+| Option | Description | Default |
+|--------|-------------|---------|
+| **Run Context** |
+| `-m`, `--modulepath FILEPATHS` | List of directories containing modules, separated by `:`. <br> Directories are case-sensitive. |
```

Comment:
Not related to this PR, but do the `global_config_setters` actually do anything for plan convert?

Reply:
https://github.com/puppetlabs/bolt/blob/52645942d5812a4f70a79f7d9a675acce44d0ca9/lib/bolt/bolt_option_parser.rb#L13